packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -666,10 +675,17 @@ export default class SortedTable extends Component {
                 selectedItemsIds[method](visibleItems[i].id)
               }
             })
-          : selectedItemsIds[method](item.id),
-    })
+          : selectedItemsIds[method](item.id)
+      selectedItems = filter(visibleItems, item =>
+        _selectedItemsIds.has(item.id)
+      )
```

Comment:
Why do you filter?

Reply:
To take the selected items

Reply:
I didn't notice that one collection was IDs and the other one was objects. Fine :+1: